allium/lib/seo.py

```
@@ -0,0 +1,127 @@
+        return path[:-10]
+    if path == "index.html":
+        return "./"
+    return path[:-5]
+
+
+def rewrite_internal_html_links(output_dir):
+    """Rewrite internal ``.html`` hrefs to their public clean-route forms."""
+    output_path = Path(output_dir)
+    changed_files = changed_links = 0
+
+    def replacement(match):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `replacement`

(ANN202)